### PR TITLE
Add repology.org to readme

### LINKED ISSUE #226: Add repology badge?
Do we want to add a badge like the one below to the readme? They also have several other badges available here https://repology.org/project/ddclient/badges.

### PR DESCRIPTION
**Pull request by SuperSandro2000**
Closes #226

### CODE REVIEW

**Comment by rhansen** [comment]
Maybe have subsections like this:

## INSTALLATION

### Distribution Package

The easiest way to install ddclient is to install a package offered by your operating system. Packages are available for the following distributions:

badge goes here

### Manual Installation

existing instructions go here

**Comment by SuperSandro2000** [comment]
I just copied your text.

File: README.md [hunk 1]
```diff
@@ -62,6 +62,16 @@ See https://github.com/ddclient/ddclient/releases
 
 ## INSTALLATION
 
+<img align="right" src="https://repology.org/badge/vertical-allrepos/ddclient.svg">
+
+### Distribution Package
+
+The easiest way to install ddclient is to install a package offered by your operating system. Packages are available for the following distributions:
+
+badge goes here
```
Comment (rhansen) [inline README.md:71]:
```suggestion
### Distribution Package

The easiest way to install ddclient is to install a package offered by your
operating system. Packages are available for the following distributions:

<img align="right" src="https://repology.org/badge/vertical-allrepos/ddclient.svg">
```

Comment (SuperSandro2000) [reply README.md:71]: I the badge shouldn't be that far down.

File: README.md [hunk 2]
```diff
@@ -62,6 +62,14 @@ See https://github.com/ddclient/ddclient/releases
 
 ## INSTALLATION
 
+<img align="right" src="https://repology.org/badge/vertical-allrepos/ddclient.svg">
+
+### Distribution Package
+
+The easiest way to install ddclient is to install a package offered by your operating system. Packages are available for the following distributions:
```
Comment (rhansen) [inline README.md:69]:
```suggestion
### Distribution Package

<img align="right" src="https://repology.org/badge/vertical-allrepos/ddclient.svg">
The easiest way to install ddclient is to install a package offered by your
operating system. See the image to the right for a list of distributions with a
ddclient package.
```

Comment (SuperSandro2000) [reply README.md]: I don't know about this. It kinda looked better when it was right under the installation headline and we didn't waste some whitespace. The badge is just to big tbh.

**Pull request merged**